packages/wallet/backend/tests/account/controller.test.ts

```diff
@@ -0,0 +1,255 @@
+              balance: 777
+            }
+          ] as Partial<RapydAccountBalance>
+        })
+      }
+    }
+    Reflect.set(accountService, 'deps', accountServiceDepsMocked)
+
+    const accountControllerDepsMocked = {
+      accountService
+    }
+    Reflect.set(accountController, 'deps', accountControllerDepsMocked)
+  })
+
+  beforeEach(async (): Promise<void> => {
+    await createUser()
+    await createReqRes()
+  })
+
+  afterAll(async (): Promise<void> => {
+    appContainer.stop()
+    knex.destroy()
+  })
+
+  afterEach(async (): Promise<void> => {
+    await truncateTables(knex)
+  })
+
+  describe('createAccount', (): void => {
+    it('should create Account for User for a given Asset', async (): Promise<void> => {
+      req.body = {
+        name: accountName,
+        assetId: mockedAsset.id
+      }
+      await accountController.createAccount(req, res, next)
+      expect(res.statusCode).toBe(200)
+      expect(res._getJSONData()).toMatchObject({
+        message: 'SUCCESS',
+        data: {
+          name: accountName
+        }
+      })
+    })
+  })
+  describe('listAccounts', (): void => {
+    beforeEach(async (): Promise<void> => {
+      await createMockAccount()
+      await createReqRes()
+    })
+    it("should list User's accounts (1 account, after creation)", async (): Promise<void> => {
+      await accountController.listAccounts(req, res, next)
+      expect(res.statusCode).toBe(200)
+
+      const jsonData = res._getJSONData()
+      expect(jsonData.message).toEqual('SUCCESS')
+      expect(jsonData.data.length).toEqual(1)
+      expect(jsonData.data[0]).toMatchObject({
+        name: accountName
+      })
+    })
+  })
+
+  describe('getAccountById', (): void => {
+    beforeEach(async (): Promise<void> => {
+      await createMockAccount()
+      await createReqRes()
+    })
+    it("should get User's Account by id", async (): Promise<void> => {
+      req.params = {
+        id: createdAccount.id
+      }
+
+      await accountController.getAccountById(req, res, next)
+      expect(res.statusCode).toBe(200)
+      expect(res._getJSONData()).toMatchObject({
+        message: 'SUCCESS',
+        data: {
+          id: createdAccount.id,
+          name: accountName
+        }
+      })
+    })
+    it("should NOT get User's Account by id, when it's not found", async (): Promise<void> => {
+      req.params = {
+        id: faker.string.uuid()
+      }
+
+      try {
+        await accountController.getAccountById(req, res, next)
+      } catch (err) {
+        expect((err as BaseError).statusCode).toEqual(404)
+      }
+    })
+  })
+  describe('fundAccount', (): void => {
+    beforeEach(async (): Promise<void> => {
+      await createMockAccount()
+      await createReqRes()
+    })
+    it("should fund User's Account", async (): Promise<void> => {
+      req.body = {
+        accountId: createdAccount.id,
+        amount: 5.7
+      }
+
+      await accountController.fundAccount(req, res, next)
```

Comment:
leave an empty line between the method and checking the expected values for better readability